core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.directive.html

```diff
@@ -116,6 +116,13 @@
       </a>
     </li>
 
+    <li ng-class="{'active': $ctrl.currentUrl === '/teach'}">
```

Comment:
Mistake? It should be `{'active': $ctrl.currentUrl === 'welcomeguidepage'}`

Reply:
Done!